cromulent/vocab.py

```diff
@@ -314,10 +313,12 @@ def register_instance(name, data):
 
 	"primary": {"parent": Type, "id": "300404670", "label": "Primary"},
 	"artwork": {"parent": Type, "id": "300133025", "label": "Artwork"},
+
+
 	"public collection": {"parent": Type, "id": "300411912", "label": "Public Collection"},
 	"style of": {"parent": Type, "id": "300404285", "label": "Style Of"},
 	"computer generated": {"parent": Type, "id": "300202389", "label": "Computer Generated"},
-	"gender issues": {"parent": Type, "id": "300233686", "label": "Gender Issues"},
+	"gender issues": {"parent": Type, "id": "300233686", "label": "Gender"},
```

Comment:
Why the difference between the key and the label?

Reply:
Good question, and good somewhat inadvertent catch ... this isn't the Gender type, which is `300055147` and managed as a pseudo-class.  Reverted!